templates/edit.html

```diff
@@ -31,15 +31,40 @@ <h1>{{ object.name.get_value }}</h1>
             {% block form_content %}{% endblock %}
         </div>
         <div class="col-sm-6" id="source-wrapper-anchor">
-            <h2>{% trans "Edit sources" %}</h2>
+            <h2>{% trans "Edit sources and confidence levels" %}</h2>
             <div class="row bg-info" id="source-wrapper">
                 <div class="col-sm-12">
+                    <div id="confidence-picker-wrapper">
+                        <!-- This widget is rendered by EJS below -->
```

Comment:
This PR makes a lot of adjustments to existing EJS (Embedded JavaScript) templates. If you haven't used EJS before on a project, you might want to take a second to read their [Getting Started and Docs](https://ejs.co/#install).